<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>EventWidget</class>
 <widget class="QWidget" name="EventWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="2" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLabel" name="voiceMailCountLabel" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QToolButton" name="voiceMailButton" >
       <property name="toolTip" >
        <string>Click here to call your voice mail</string>
       </property>
       <property name="text" >
        <string>New Voice Mail:</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/profilebar/event/voicemessage_new.png</iconset>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QToolButton" name="missedCallButton" >
       <property name="toolTip" >
        <string>Click here to see missed call(s)</string>
       </property>
       <property name="text" >
        <string>Missed Call(s):</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/profilebar/event/call_missed.png</iconset>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="missedCallCountLabel" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>47</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>missedCallButton</tabstop>
  <tabstop>voiceMailButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
